src/main/java/games/strategy/engine/lobby/client/login/LobbyLogin.java

```diff
@@ -158,7 +162,11 @@ public void notifyFailedLogin(final String message) {
               final Map<String, String> props = new HashMap<>();
               props.put(LobbyLoginValidator.REGISTER_NEW_USER_KEY, Boolean.TRUE.toString());
               props.put(LobbyLoginValidator.EMAIL_KEY, createAccount.getEmail());
+              // TODO: Don't send the md5-hashed password once the lobby is updated
```

Comment:
If this is being removed now, this makes it hard for us developers to test if the lobby still works, we can of course remove this once we have an updated test lobby!